marbles/core/tests/test_main.py

```diff
@@ -79,10 +79,10 @@ def __run_cmd(self):
                 pth.flush()
 
             coverage_config = os.path.join(core_dir, '.coveragerc')
-            env = {'COVERAGE_PROCESS_START': coverage_config}
+            env = {**os.environ, **{'COVERAGE_PROCESS_START': coverage_config}}
```

Comment:
Just a minor thing, only `os.environ` needs the `**`, the coverage key can just be inline: `{'COVERAGE_PROCESS_START': coverage_config, **os.environ}`